website/source/docs/drivers/docker.html.md

```diff
@@ -88,9 +88,10 @@ The `docker` driver supports the following configuration in the job spec.  Only
 * `extra_hosts` - (Optional) A list of hosts, given as host:IP, to be added to
   `/etc/hosts`.
 
-* `force_pull` - (Optional) `true` or `false` (default). Always pull latest image
```

Comment:
not sure about this change - latest here can a bit ambiguous and we just mean most recent/updated not `latest` tag.

Reply:
Yeah I just stared at it for a good few mins thinking about if it should be “most recent image:tag”.

I agree latest as a term is overloaded